pkg/providers/imagefamily/image_test.go

```diff
@@ -199,3 +224,10 @@ var _ = Describe("Image ID Parsing", func() {
 		Entry("empty image id should not parse", "badimageid", "", "", "", true),
 	)
 })
+
+var _ = Describe("ImageFamily Name", func() {
+	It("should return the correct image family name", func() {
+		Expect(imagefamily.Ubuntu2204{}.Name()).To(Equal(v1alpha2.Ubuntu2204ImageFamily))
+		Expect(imagefamily.AzureLinux{}.Name()).To(Equal(v1alpha2.AzureLinuxImageFamily))
+	})
+})
```

Comment:
This seems unnecessary. Other tests will break if this does not hold.

Reply:
i agree was just farming for 100% coverage

Reply:
To be removed?